collector/netframework_clrexceptions.go

```diff
@@ -0,0 +1,132 @@
+// returns data points from Win32_PerfRawData_NETFramework_NETCLRExceptions
+// <add link to documentation here> - Win32_PerfRawData_NETFramework_NETCLRExceptions class
+package collector
+
+import (
+	"log"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	Factories["netframework_clrexceptions"] = NewWin32_PerfRawData_NETFramework_NETCLRExceptionsCollector
+}
+
+// A Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector is a Prometheus collector for WMI Win32_PerfRawData_NETFramework_NETCLRExceptions metrics
+type Win32_PerfRawData_NETFramework_NETCLRExceptionsCollector struct {
+	NumberofExcepsThrown       *prometheus.Desc
+	NumberofExcepsThrownPersec *prometheus.Desc
+	NumberofFiltersPersec      *prometheus.Desc
+	NumberofFinallysPersec     *prometheus.Desc
+	ThrowToCatchDepthPersec    *prometheus.Desc
```

Comment:
The struct field names can have some more descriptive names, rather than just the WMI field name, so `NumberOfExceptionsThrown`, `ThrowToCatchDepth`, and so on.

The `PerSec` suffix can always be removed, and is even misleading (in the PerfRaw counters, they are actually the number of times the counter has been incremented, not the rate)